### DISCUSSION TITLE
CAM16 / CAT16 / CAM16-UCS

### DISCUSSION THREAD

**Discussion by danburzo** [Evercoder/culori#133] [Ideas]
https://github.com/d3/d3-cam16
https://github.com/connorgr/d3-cam02
http://gramaz.io/d3-cam02/

**Comment by danburzo** [comment]
https://github.com/njsmith/colorspacious/issues/10
https://github.com/ksmet1977/luxpy/tree/master/luxpy

**Comment by buddye** [comment]
I used the typescript port of colorio https://github.com/reinventing-wheels/cam16

colorio by Nico Schlömer who I believe published the cam16 fixes paper

Also, +1 for CAM16-UCS, will hopefully be able to help soon, thanks!

**Comment by danburzo** [comment]
Awesome, thanks for the recommendations! The Nico Schlömer paper looks good.

**Comment by orionrush** [comment]
See also:
- https://github.com/baskerville/ciecam02

- https://github.com/eeeps/cam02-color-schemer
Built on Connor Gramazio’s implementation of CIECAM02 in D3

- https://observablehq.com/@jrus/cam16
A notebook implementation of the CAM16 color appearance model - author notes typos in Schlömer's paper:
https://github.com/nschloe/note-on-cam16/blob/gh-pages/main.pdf
